File: Java/operationsApplication/TemperatureApp.java
package operationsApplication;

import operationsApplication.PerformTemperatureConversion;

public class TemperatureApp 
{
	
	public static void main(String[] args) 
	{
		
		System.out.println("--------- TEMPERATURE CONVERSION APP ---------\n");
		
		System.out.println("CELCIUS TO FAHRENHEIT");
		PerformTemperatureConversion operation1 = new PerformTemperatureConversion();
		operation1.performCtF();
		
		System.out.println("\n\n----------------------------------------------");
		
		System.out.println("\nFAHRENHEIT TO CELCIUS");
		operation1.performFtC();
	}
	
}

File: TemperatureApp.cs
using System;

using static OperationsApp.PerformTemperatureConversion;

namespace OperationsApp
{
    class TemperatureApp
    {
        static void Main(string[] args)
        {
            Console.WriteLine("--------- TEMPERATURE CONVERSION APP ---------\n");

            Console.WriteLine("CELCIUS TO FAHRENHEIT");
            PerformTemperatureConversion operation1 = new PerformTemperatureConversion();
            operation1.performCtF();

            Console.WriteLine("\n----------------------------------------------");

            Console.WriteLine("\nFAHRENHEIT TO CELCIUS");
            operation1.performFtC();

            Console.WriteLine("\n----------------------------------------------");
        }
    }
}
